<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PointsEditorDialog</class>
 <widget class="QDialog" name="PointsEditorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>224</width>
    <height>299</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit points</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableWidget" name="pointsTW">
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="addPB">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deletePB">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="savePB">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelPB">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>savePB</sender>
   <signal>clicked()</signal>
   <receiver>PointsEditorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>31</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>54</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelPB</sender>
   <signal>clicked()</signal>
   <receiver>PointsEditorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>144</x>
     <y>274</y>
    </hint>
    <hint type="destinationlabel">
     <x>166</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
